src/test/triggers/dropdowns/getContentTypes.test.ts

```diff
@@ -0,0 +1,70 @@
+import { createAppTester } from 'zapier-platform-core';
+import App from '../../../index';
+import * as nock from 'nock';
+import { mockBundle } from '../../utils/mockBundle';
+import getContentTypes from '../../../triggers/dropdowns/getContentTypes';
+import { Contracts, DeliveryClient } from '@kontent-ai/delivery-sdk';
+
+const appTester = createAppTester(App);
+nock.disableNetConnect();
+
+afterEach(() => nock.cleanAll());
+
+describe('getContentTypes', () => {
+  it('returns all content types returned by the Delivery API', async () => {
+    const bundle = mockBundle;
+
+    const expectedRequest = new DeliveryClient({
+      projectId: bundle.authData.projectId,
+      previewApiKey: 'previewAPIKey',
+    })
+      .types()
+      .queryConfig({ usePreviewMode: true });
```

Comment:
is preview necessary in this context?

Reply:
No really, but I didn't want to introduce too many changes. I will submit a separate issue for this.